services/frontend/src/components/SignIn.vue

```diff
@@ -19,7 +19,7 @@
     />
     <a  
       class="input-element-standard button link"
-      href="https://www.google.com"
+      href="https://www.w3schools.com"
```

Comment:
Just out of curiosity, why did you change that?

Reply:
privacybadger said that this was a tracker, and i want to have no trackes for now :D